nemoguardrails/http/transport.py

```
@@ -0,0 +1,141 @@
+            follow_redirects: Whether an owned client follows redirects.
+
+        Raises:
+            ValueError: If the configured timeout is not positive.
+        """
+
+        if timeout is not None and timeout <= 0:
+            raise ValueError("HTTP timeout must be greater than zero")
+        self._owns_client = client is None
+        self._timeout = timeout if self._owns_client else None
+        self._client = client or httpx.AsyncClient(
+            timeout=None,
```

Ruff (0.15.21)

[error] Probable use of `httpx` call with timeout set to `None`

(S113)